addons/website_slides/static/src/xml/slide_management.xml

```diff
@@ -16,6 +16,7 @@
                     <label for="section_name" class="col-sm-3 col-form-label">Section name</label>
                     <div class="col-sm-9">
                         <input type="text" class="form-control" name="name" id="section_name" required="required"/>
+                        <p id="name-required" class="text-danger mt-1 mb-0 d-none">Please fill in this field</p>
```

Comment:
I would not bother doing that.

There is only one field so it's pretty clear you have to fill it in.
Also, it's not consistent with the "slide upload" form validation that only shows a red border for invalid fields.